website/source/docs/internals/architecture.html.md

```diff
@@ -16,6 +16,7 @@ page documents the system architecture.
 [glossary](/docs/glossary.html) of terms to help
 clarify what is being discussed.
 
+You can apply the architecture concepts in this document to a local Consul cluster by following this [Getting started guide](https://learn.hashicorp.com/consul/getting-started/join?utm_source=consul.io&utm_medium=docs).
```

Comment:
🤔 I'm not sure this is the right thing to link from here exactly. If we had our "server and client tour" guide it would be perfect here but alas. Maybe the reference architecture might b a good resource instead?